src/Icon/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import styles from './Icon.scss';
+
+function Icon(props) {
+  return (
+    <div>
+      <span
+        id={props.id}
+        className={classNames(props.className)}
+        aria-hidden={props.hidden}
+      />
+      { props.srText &&
+        <span className={classNames(styles['sr-only'])}>
+          {props.srText}
+        </span>
+      }
+    </div>
+  );
+}
+
+Icon.propTypes = {
+  id: PropTypes.string.isRequired,
+  className: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
If this is an array of strings, should the name indicate multiple values? Obviously, there's a conflict with `classNames` but maybe there's some other name that could make sense?

Reply:
`className` is the name for the string array format on all other Paragon components that take that as a prop.

Reply:
Consistency is important but we could consider in the future

1. Renaming the `classNames` import within the file and thus, resolving the naming conflict
2. Using something like `classes` instead of `className`